man/adoc/bpftrace.adoc

```diff
@@ -3013,50 +3013,52 @@ iter:task_file:/sys/fs/bpf/files {
  */
 ----
 
-[#probes-kfunc]
-=== kfunc and kretfunc
+[#probes-fentry]
+=== fentry and fexit
 
 .variants
-* `kfunc[:module]:fn`
 * `fentry[:module]:fn`
-* `kretfunc[:module]:fn`
+* `kfunc[:module]:fn`
```

Comment:
Why do we want to leave these references in the adoc? I would probably just put a little note that these are deprecated but not offer them as valid variants.

Reply:
That note is already there few lines below, so I guess that we can just drop this.